### ISSUE TITLE
Bad label for DISPLAY_CONTAINER_NAME in docker-compose-v3-video.yml for Opera

### ISSUE THREAD

**Issue by wobal** [SeleniumHQ/docker-selenium#1161]
## 🐛 Bug Report

In the file **docker-compose-v3-video.yml** there is firefox instead of opera for the **DISPLAY_CONTAINER_NAME**

```
  opera_video:
    image: selenium/video:ffmpeg-4.3.1-20201208
    volumes:
      - /tmp/videos:/videos
    depends_on:
      - opera
    environment:
      - DISPLAY_CONTAINER_NAME=firefox
      - FILE_NAME=opera_video.mp4
```

**Comment by diemol** [comment]
@wobal, would you like to send a PR to fix that?

**Comment by diemol** [comment]
Fixed via #1163 
Thank you, @wobal!
